gammapy/maps/wcs/geom.py

```diff
@@ -397,8 +397,8 @@ def create(
         if npix is None:
             width = cast_to_shape(width, shape, float)
             npix = (
-                np.rint(width[0] / binsz[0]).astype(int),
-                np.rint(width[1] / binsz[1]).astype(int),
+                np.floor(width[0] / binsz[0] + 0.5).astype(int),
+                np.floor(width[1] / binsz[1] + 0.5).astype(int),
```

Comment:
Is it really needed here? How dos it change the behavior?

Reply:
No indeed, this was not part of the original PR, but I was wondering if this can affect the mismatch in number of events @facero is seeing. I could not find a simple way to test this.

Reply:
I reverted this change since it does not affect (as expected)